MSAL/src-internal/ios/MSALLegacySharedMSAAccount.m

```diff
@@ -124,6 +125,27 @@ - (BOOL)matchesParameters:(MSALAccountEnumerationParameters *)parameters
 
 #pragma mark - Updates
 
+- (BOOL)updateAccountWithMSALAccount:(id<MSALAccount>)account
```

Comment:
nit: similar logic in MSALLegacySharedADALAccount.

Reply:
👍

Reply:
Addressed